serenity/serenity.hpp

```diff
@@ -42,7 +41,11 @@ class Producer : public BusSocket {
   virtual ~Producer() {}
 
   Try<Nothing> addConsumer(Consumer<T>* consumer) {
-    consumers.push_back(consumer);
+    if (consumer != nullptr) {
+      consumers.push_back(consumer);
+    } else {
+      LOG(ERROR) << "You tried to add nullptr as a consumer";
```

Comment:
Nice guard, but would you mind updating the message to anonymous third-person?  E.g. "Consumer must not be null".